IntendedFor.py

```diff
@@ -0,0 +1,67 @@
+
+#!/usr/bin/python
+import json
+import os
+from glob import glob
+import numpy
+
+"""
+This script is designed to determine which field maps apply to discrete fMRI scans
```

Comment:
This looks very similar to `get_fieldmap` implemented in pybids https://github.com/INCF/pybids/tree/master/bids/grabbids#get-a-fieldmap-files-intended-for-a-given-map. Maybe we should use that one?

Reply:
Hmm. It has been a while since I looked at the get_fieldmap function implemented in pybids so perhaps it has changed, but doesn't the get_fieldmap function just retrieve the fieldmap in which the fmriname is within the "IntendedFor" field? I was under the impression that the "IntendedFor" field does not automatically generate through the BIDS conversion, thus it has to be added manually. 

What am I missing?

Reply:
Correct. Maybe I misunderstood what your function does. If it's part of a conversion process this repo might not be the best place for it. Maybe it would be better to put it in pybids?

Reply:
I can see why you may have been confused. The comments I provide I very unclear. Obviously I created them when I did not understand what the script was doing myself, and forgot to come through and change them. It is added the "IntendedFor" fields to the fmap json sidecars, so yes I suppose it has more to do with the BIDS conversion. I can propose incorporation to pybids, but perhaps as a catch-all let's leave IntendedFor..py as is for now.